app/code/core/Mage/Checkout/Block/Cart/Sidebar.php

```diff
@@ -293,16 +298,6 @@ public function getCacheTags()
         );
     }
 
-    /**
-     * Get form key
-     *
-     * @return string
-     */
-    public function getFormKey()
```

Comment:
can we remove this method?

Reply:
Y, it is same as in parent `Mage_Core_Block_Abstract::getFormKey()`

Reply:
true